### PR TITLE
#164798174 Sharing of articles

### PR DESCRIPTION
**Pull request by felixkiryowa**
#### What does this PR do?

- Sharing of articles 

#### Description of Task to be completed?
-  A user should be able to share articles on social media and via email.

#### How should this be manually tested?

- Fetch this branch `git fetch origin   ft-share-articles-164798174`
- Checkout to the branch `git checkout    ft-share-articles-164798174 `
- Run `npm install`
- Run `npm run start:dev`
- Login into the application
- Click on the share icon on any article and share it with other users.

#### What are the relevant pivotal tracker stories?
[#164798174](https://www.pivotaltracker.com/story/show/164798174)
Screenshot 2019-05-22 at 19 17 40
Screenshot 2019-05-22 at 19 18 11
Screenshot 2019-05-22 at 19 20 24
Screenshot 2019-05-23 at 11 07 10

Screenshot 2019-05-23 at 12 53 13

### CODE REVIEW

File: src/components/Articles/articlesIcons.jsx [hunk 1]
```diff
@@ -1,23 +1,50 @@
...
+          </a>
+          <a className="dropdown-item" href={articleShare.mailshare}>
+            <i className="fas fa-envelope-square envelop-icon" />
+            Mail
+          </a>
+        </div>
+      </div>
+      &nbsp;
+      &nbsp;
+      <a href="hfjghjg">
```
Comment (ezrogha) [inline src/components/Articles/articlesIcons.jsx]: what does `hfjghjg` mean?
 Please use a more descriptive className

File: src/components/Articles/articlesIcons.jsx [hunk 2]
```diff
@@ -1,23 +1,50 @@
...
+        <i className="far fa-comment" />
+      </a>
+      &nbsp;
+      &nbsp;
+      <a href="hjgjghg">
```
Comment (ezrogha) [inline src/components/Articles/articlesIcons.jsx]: Please use a more descriptive  name

File: src/components/Articles/articlesIcons.jsx [hunk 3]
```diff
@@ -1,23 +1,50 @@
...
-    &nbsp;
 
-  </div>
-);
+const articlesIcons = ({ article }) => {
+  const articleShare = ShareLinks(article, article.slug);
+  return (
+    <div className="icons">
+      <div className="btn-group dropleft">
+        <a href="hhgfhg" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```
Comment (ezrogha) [inline src/components/Articles/articlesIcons.jsx]: Please use a more descriptive name

File: src/components/Articles/articlesIcons.jsx [hunk 4]
```diff
@@ -1,23 +1,50 @@
...
+        <i className="far fa-comment" />
+      </a>
+      &nbsp;
+      &nbsp;
+      <a href="hjgjghg">
+        <i className="far fa-thumbs-up pull float-right" />
+      </a>
+      &nbsp;
+      &nbsp;
+      <a href="hgjghgj">
```
Comment (ezrogha) [inline src/components/Articles/articlesIcons.jsx]: Please use a more descriptive name

**Review by ezrogha — DISMISSED** [review]
great work with the feature @felixkiryowa,
Kindly work on the feedback

**Review by hadijahkyampeire — APPROVED** [review]
looks good well done

**Pull request merged**